README.md

```diff
@@ -296,7 +296,20 @@ Other interesting functions you can use are: `randBytes`, `htpasswd`, `encryptAE
 Sometimes you have tests that work well for some platform combinations, e.g. Apache + modsecurity2, but fail for others, e.g. NGiNX + modsecurity3. Taking that into account, you can override test results using the `testoverride` config param. The test will be skipped, and the result forced as configured.
 
 Tests can be altered using four lists:
-- `input` allows you to override global parameters in tests. An example usage is if you want to change the `dest_addr` of all tests to point to an external IP or host
+- `input` allows you to override global parameters in tests. The following ones can be overridden:
+  - `dest_addr`: allows to point to an external IP or host
+  - `override_empty_host_header`: enabling this flag, empty Host headers will be replaced with `dest_addr`
+  - `port`: overrides the port number
+  - `protocol`: overrides the protocol
+  - `uri`: overrides the uri
+  - `version`: overrides the HTTP version. E.g. "HTTP/1.1"
+  - `headers`: overrides headers, the format is a map of strings
+  - `method`: overrides the method used to perform the request
+  - `data`: overrides data sent in the request
+  - `savecookie`: 
```

Comment:
Not yet implemented. @fzipi any plans to do that?

Reply:
Many thanks for the suggested changes Max! Being not implemented I temptatively removed it from the list

Reply:
I think we need to uncomment https://github.com/coreruleset/go-ftw/blob/94ab311941820255097b13fe53c6f7276e19613a/ftwhttp/request.go#L165-L170 and provide some tests.

Reply:
Ah yes. In that case @M4tteoP, I propose to drop `savecooke` for now, as you did. We can add it to the documentation once cookie handling has been implemented.